inductiva/fluids/scenarios/fluid_tank/output.py

```diff
@@ -0,0 +1,61 @@
+"""Post-processing and visualization utilities of the fluid tank scenario."""
+
+import os
+from typing import Optional
+
+import xarray as xr
+
+from inductiva.types import Path
+from inductiva.utils import files, visualization
+
+
+class FluidTankOutput:
```

Comment:
Later on we need to standardize the methods of other SPH scenarios with this one. Although this one contains some external dependencies that I am not aware if they will get misconfigured.

Reply:
Agreed.